src/index.ts

```diff
@@ -20,10 +21,14 @@ export default async function createAuth0Client(options: Auth0ClientOptions) {
       'For security reasons, `window.crypto` is required to run `auth0-spa-js`.'
     );
   }
+  // safari 10 compat
+  if ((<any>window.crypto).webkitSubtle) {
```

Comment:
The guidance at https://webkit.org/blog/7790/update-on-web-cryptography/ suggests you use subtle first and only fall back to webkitSubtle if that fails.  It looks like this is reversed?  If it exists it uses it over subtle?  They specifically call out that as bad behavior on that page.

Reply:
fixed